src/Config/Processor/BuilderProcessor.php

```diff
@@ -45,6 +48,14 @@
     public static function process(array $configs): array
     {
         foreach ($configs as &$config) {
+            if (isset($config['config']['builders']) && \is_array($config['config']['builders'])) {
+                if (!isset($config['config']['fields'])) {
```

Comment:
We could merge if not empty and set to builder result otherwise, this remove a useless array merge.

Reply:
Fixed